include/wil/cppwinrt_helpers.h

```diff
@@ -212,6 +212,48 @@ namespace wil::details
 #endif // __WIL_CPPWINRT_MICROSOFT_UI_DISPATCHING_HELPERS
 /// @endcond
 
+#if defined(WINRT_Windows_Foundation_H) && !defined(__WIL_CPPWINRT_WINDOWS_FOUNDATION_HELPERS)
+#define __WIL_CPPWINRT_WINDOWS_FOUNDATION_HELPERS
+namespace Windows::Foundation

```

Comment:
Do we need to do the `ABI` namespace mumbo gumbo here?

Reply:
Answering my own question. This isn't a WinRT/MIDLRT generated header, so that shouldn't be a concern unless someone includes it within a `namespace ABI` guard